VideoLocker/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -486,9 +491,6 @@ public synchronized void playOrPrepare(String path, int seekTo, String title,
             exitFullScreen();
         }
 
-        // clear all errors
-        clearAllErrors();
```

Comment:
This call here was redundant and clearing errors here will still be redundant, as all the errors are cleared once we go into `onPrepared`, which is actually the right place to do it.